sdk/ai/ai-projects/test/public/utils/createClient.ts

```diff
@@ -89,6 +89,13 @@ const recorderEnvSetup: RecorderStartOptions = {
 export async function createRecorder(context: VitestTestContext): Promise<Recorder> {
   const recorder = new Recorder(context);
   await recorder.start(recorderEnvSetup);
+  if (isPlaybackMode()) {
+    // To handle the unpredicable content-type header (in different environments?)
+    // The following matcher is to avoid the mismatch between the request and the recording:
+    // Header differences:
+    //    <Content-Type> values differ, request <application/json>, record <application/json; charset=UTF-8>
+    await recorder.setMatcher("CustomDefaultMatcher", { excludedHeaders: ["Content-Type"] });
+  }
```

Comment:
can we treat `<application/json>` and `<application/json; charset=UTF-8>` the same at the Test proxy level to avoid the mismatch?

@scbedd @jeremymeng